zerver/tests/test_subs.py

```diff
@@ -1469,7 +1469,7 @@ def test_gather_subscriptions(self):
             if not sub["name"].startswith("stream_"):
                 continue
             self.assertTrue(len(sub["subscribers"]) == len(users_to_subscribe))
-        self.assert_length(queries, 4, exact=True)
+        self.assert_length(queries, 9, exact=True)
```

Comment:
In my comment on a previous diff, I meant that you should not have to increase these query limits to add your feature